day_7/caesar_cipher.rb

```diff
@@ -0,0 +1,50 @@
+# implements the Caesar Cipher on any message for a positive shift
```

Comment:
Fantastic job challenging yourself on the Cipher challenge Allison! I love that you took user input and encapsulate this challenge within a class. If you have time this weekend I encourage you to time box the other challenges to about and hour or two to keep building on these ruby skills. You'll see a lot of timed challenges at Turing and during the interview process, so time boxing is a great way to practice! You did great on the capstone! Good luck in mod-1!